src/NavLink.php

```diff
@@ -75,6 +85,49 @@ public static function to(
         return new self($active, $attributes, $encodeLabel, $disabled, $label, $url, $urlAttributes, $visible);
     }
 
+    /**
+     * Creates a {@see NavLink} instance for a tab.
+     *
+     * @param string|Stringable $label The label of the tab.
+     * @param string|Stringable $content The content of the tab pane.
+     * @param bool $active Whether the tab is active.
+     * @param bool $disabled Whether the tab is disabled.
```

Comment:
I don't see it in the list of method arguments.

Reply:
done.